gitlab-ls.py

```diff
@@ -32,36 +31,14 @@ def to_completion_item(self):
         )
 
 
-@dataclass_json
-@dataclass
-class GitlabMergeRequest:
```

Comment:
Good idea to merge merge requests and issues together since they are the same. Maybe  name the resulting class could have a more generic name, like `GitlabObject` ?

Reply:
done